cpp/velox/compute/VeloxBackend.cc

```diff
@@ -208,50 +178,35 @@ void VeloxBackend::init(const std::unordered_map<std::string, std::string>& conf
   }
 
   std::unordered_map<std::string, std::string> s3Config({});
```

Comment:
remove variable `s3Config` as it's not used anymore.

Reply:
Oh, missed your comments. Let's remove it in next PR